lib/vfs/src/lib.rs

```diff
@@ -106,7 +142,11 @@ impl OpenOptionsConfig {
     }
 }
 
-// TODO: manually implement debug
+impl fmt::Debug for OpenOptions {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        self.conf.fmt(f)
```

Comment:
I doubt anyone will notice, but this will print out `OpenOptionsConfig { ... }` instead of `OpenOptions { ... }`

Reply:
Yeah I noticed, but I don't think it's that important